Packs/CommonScripts/Scripts/IsEmailAddressInternal/IsEmailAddressInternal.yml

```diff
@@ -41,3 +41,4 @@ scripttarget: 0
 fromversion: 5.0.0
 tests:
 - IsEmailAddressInternal_Test
+- TestIsEmailAddressInternal
```

Comment:
was this missed?

Reply:
It's an old TPB and it didn't run in the previous build [here](https://github.com/demisto/content/pull/30897), so it didn't fail and we didn't catch the bug